src/ui.tsx

```diff
@@ -100,6 +114,102 @@ export class UserInterface extends Component<Props, State> {
 
   private imageFileInfo: ImageFileInfo | null;
 
+  private canvasContainer: HTMLDivElement;
+
+  toolTips: ToolTips[] = [
+    {
+      name: "Select",
+      icon: `./src/assets/select-icon.svg`,
+      shortcut: "V",
```

Comment:
I'm not sure what we want to do about these, as we _also_ have keybinding set elsewhere...and these don't seem to work? 

Ideally these would all be in one place, possibly with default shortcuts and then use the customisation we have now on top?

Reply:
The tooltips and icons probably belong in a _toolbox_, a level below ui.tsx?

Reply:
> Some further comments @ChrisBaidoo, now that the visuals are working:
> 
> * [x]  The button background is not the same colour as the toolbar background, so I can see a faint white circle behind each button - is this something we can fix? Probably worth making sure both buttons and toolbars are using the same palette colour?
> * [x]  When I hover over a button, the white circle changes to green but a faint grey square appears too - is this to do with those hardcoded hexes in the code body? If we move everything over to using the palette colours this should prevent this sort of thing right?
> * [x]  The paintbrush colour is still pink, could we make this our new primary green colour?
> * [x]  Same with splines and spline nodes - could they be made to match our colour palette?
> * [x]  I love the little labels popover... except I can't actually add my label? I just have a test area and can type but there's no 'add' button and pressing Return doesn't do anything either?
> * [x]  When I click 'Select' - I expected that to put me into select mode, which currently has the keyboard shortcut of Ctrl+A (not V as per the tool tip); is that what this button is meant to do?
> * [ ]  Also, after clicking around for a little while I managed to break it so that the paintbrush stopped working - no idea how though, so we should keep an eye out for this.
> * [x]  Is it possible to turn scroll bars off somehow? And do this without obscuring parts of the image? In my browser I get an up-down scrollbar all the time and when I go to the bottom right corner to turn the minimap on the left-right scroll bar appears?
> * [ ]  I thought we turned painting outside the image off? @SilviaZeta? @cooper667? Or @philipjackson? Because I can still paint outside the image?
> * [x]  The minimap works great... however, on my browser, Ctrl+ and Ctrl- make the browser zoom in/zoom out occur, not the app zoom in/out... and this also breaks the position of the image behind the app bar...? Maybe a separate issue needs to be made for this one?

Created an issue for the Zoom in and out. #194